src/content/docs/opentelemetry/nrdot/nrdot-intro.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: New Relic Distribution of OpenTelemetry (NRDOT)
+tags:
+  - OpenTelemetry
+  - NRDOT
+metaDescription: New Relic Supported Distribution of OpenTelemetry
+freshnessValidatedDate: 2024-05-14
+---
+
+The New Relic Distribution of OpenTelemetry (NRDOT) is a New Relic-supported distribution of the OpenTelemetry project.
+It provides a set of OpenTelemetry components that are tested and supported by New Relic.
+NRDOT is designed to work seamlessly with New Relic products and services, and it is the recommended way to instrument your applications with OpenTelemetry for use with New Relic.
```

Comment:
Similar to the previous comment, this implies that NRDOT alternatives to OpenTelemetry components are available in all domains. Let's rephrase to make it clear that NRDOT is recommended when available.

```suggestion
NRDOT components are designed to work seamlessly with New Relic products and services, and when available, are the recommended way to integrate OpenTelemetry with New Relic.
```